README.md

```diff
@@ -1,6 +1,6 @@
 # Sonata Media CKEditor integration
 
-The bundle provides [SonataMediaBundle](http://sonata-project.org/bundles/media/master/doc/index.html) integration into [CKEditor](http://ckeditor.com/) for Symfony 2 projects.
+The bundle provides [SonataMediaBundle](http://sonata-project.org/bundles/media/master/doc/index.html) integration into [CKEditor](http://ckeditor.com/) for Symfony 3 projects.
```

Comment:
I suggest to just say "for Symfony projects", it's the official denomination, and it will work with Symfony 4 too.